### DISCUSSION TITLE
Is there a way to start GBM in the tray at windows startup?

### DISCUSSION THREAD

**Discussion by jazir555** [MikeMaximus/gbm#288] [Q&A] [answered]
Is there a way to get GBM to launch in the tray in the corner?

**Comment by MikeMaximus** [comment, marked as answer]
Click **File -> Settings** and enable **Start automatically at log-in** in the **Startup** section.  Also enable **Start minimized** if you'd like it to go directly to the system tray.

If you're always running GBM as Administrator for some reason you need to [follow these more advanced steps](http://mikemaximus.github.io/gbm-web/gbm_task_scheduler.html)
 to have it start automatically with Windows.
